src/app/shared/components/NavBar/index.tsx

```diff
@@ -38,7 +38,7 @@ const NavBar = async (props: NavbarProps) => {
         </div>
 
         {authenticated
-          ? <UserMenu user={user} />
+          ? <UserMenu user={user!} />
```

Comment:
oooh cool

 _In TypeScript, a postfix ! removes null and undefined from the type of an expression.

This is useful when you know, for reasons outside TypeScript's inference ability, that a variable that "could" be null or undefined actually isn't._